docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -30,14 +30,137 @@ Understanding GBD's Disease and Health Categories
+
+
+Causes
+++++++
+
+The cause hierarchy diagram for IHD and HHD is included here as a reference. 
+
+.. image:: cause_hierarchy_ihd_hhd.svg
+
+As you can see, the cause hierarchy slowly breaks down cause groups into more specific 
+diseases, moving from all non-communicable diseases to specific types of cardiovascular 
+disease. At each level (labeled), the cause hierarchy is mutually exclusive and 
+collectively exhaustive - meaning all disease and death can be placed into a group at 
+each level of the diagram. A cause is anything that causes death or disability. 
+
+Both IHD and HHD are within the more general cardiovascular disease cause (c_491). IHD and HHD 
+are "causes" within GBD. 
+
+Sequelae
+++++++++
+
+Once you have selected a specific cause - for example, IHD - you can then identify the 
+sequelae within that cause. These are all of the items listed below IHD and HHD in the 
+diagrams above. 
+
+Sequelae detail a specific medical condition associated with 
+a cause. For example, IHD has medical conditions like acute myocardial infarction 
+(a heart attack) and angina (chest pain) as associated sequelae. These are the things 
+a patient might be experiencing, or a doctor might identify. 
+
+Sequelae often contain a level of severity for that condition - for example, mild and moderate 
+angina are both sequelae within IHD. 
+
+Health States
++++++++++++++
+
+A health state is the amount of health or disability within a sequelae. This is summarized 
+through the disability weight. More information on this can be found in `this section of the cause model page <https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_model_components/causes/index.html#disability-weights>`_. A health state for a sequelae and its disability weight can be 
+found in the GBD methods appendix. 
+
+Using our above sequelae (acute myocardial infarction and angina) as an example - 
+the amount of health lost while experiencing an acute 
+myocardial infarction is quite high! You're likely hospitalized and in a lot of pain. 
+However, the health lost from angina might be less significant. You're likely at home and 
+might only experience discomfort when under stress. This affects your disability weight value. 
+
+Each sequela has exactly one health state associated with it. If you are experiencing multiple 
+symptoms, such as "headache and chest pain", your health state contains all of these losses to 
+health in one "health state". 
+
+One important note is that health states can be shared across sequelae - and even shared 
+between sequelae of different causes. For example, severe heart failure due to IHD and 
+severe heart failure due to HHD have the same amount of health loss associated with them. 
+Severe heart failure feels the same to a patient no matter the cause. Therefore, we would 
+say that the health state and therefore the disability weight is the same for both sequelae. 
+
+Impairments
```

Comment:
Here's how GBD modelers described impairments for GBD 2015 in the slide deck from the Impairments 101 HUB page:

> What is an impairment? 
•“Impairments in GBD are disorders or specific domains of functional health loss that are spread across many GBD causes as sequelae and for which there are better data to estimate the occurrence of the overall impairment than for each sequela based on the underlying cause” (2015 YLD capstone)
•TLDR: Outcome of multiple causes

Maybe it would be good to use/reference/paraphrase this description somewhere?